ansible_base/lib/redis/client.py

```diff
@@ -63,6 +64,12 @@ def connect(self, index: int = 0) -> Union[Redis, RedisCluster]:
         except (IndexError, ValueError):
             pass
 
+        for file_setting in ['ssl_certfile', 'ssl_keyfile', 'ssl_ca_certs']:
+            file = kwargs.get(file_setting, None)
+            if file:
+                if not os.access(file, os.R_OK):
```

Comment:
Minor nit: Can be combined in a single if expression:
```suggestion
            if file and not os.access(file, os.R_OK):
```